src/Spatial/Euclidean/Plane.cs

```diff
@@ -74,6 +77,30 @@ public Plane(Point3D rootPoint, UnitVector3D normal)
         {
         }
 
+        /// <summary>
+        /// Initializes a new instance of the <see cref="Plane"/> struct.
+        /// Constructs a Plane from the given points.
+        /// </summary>
+        /// <param name="points">The points</param>
+        /// <returns>Fitted <see cref="Plane"/></returns>
+        /// <remarks>this method uses SVD to fit the plane.</remarks>
+        public static Plane CreateFittedPlaneFrom(IEnumerable<Point3D> points)
+        {
+            var c = Point3D.Centroid(points);
```

Comment:
I would use more descriptive variable names, like `throughPoint` instead of just `c`.

Reply:
will do, thanks.